modules/core/src/main/java/org/apache/ignite/internal/binary/BinaryUtils.java

```diff
@@ -2827,6 +2829,34 @@ public static BinaryReaderEx reader(BinaryContext ctx,
         return new BinaryReaderExImpl(ctx, in, ldr, hnds, skipHdrCheck, forUnmarshal);
     }
 
+    /**
+     * @param ctx Context.
+     * @return Writer instance.
+     */
+    public static BinaryWriterEx writer(BinaryContext ctx) {
+        BinaryThreadLocalContext tlsCtx = BinaryThreadLocalContext.get();
```

Comment:
tls, why s on the end? This name is a little bit misleading. 

locCtx instead?